Utils/Dataflow/pyDKB/common/custom_readline.py

```diff
@@ -0,0 +1,13 @@
+def custom_readline(f, newline):
```

Comment:
Please, add docstrings here.
Although the function is not over-complicated, it would be easier to read it when you know in advance what is `f` and `newline`, and what is expected as a function result :)

Reply:
Thank you for 34c8505.

One more comment: PEP8 [suggests](https://www.python.org/dev/peps/pep-0008/#documentation-strings) to follow the [docstring conventions](https://www.python.org/dev/peps/pep-0257/).
In short, the last states:
* one-string docstrings (for obvious cases) are to look like this:
```
def function(a, b):
    """Do X and return a list."""
```
* multiline docstrings are to look like this:
```
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
```
, where the first line is a "summary", then goes a blank line, and then -- everything you would like to say about the function in addition to the summary (just like in commit logs, right?)

We (or, at least, I) do not use tools for auto-generating of project documentation (yet ;) ), but even for a human eye it looks more structured if follows the recomendations.

The PEP document also provides some recommendations about what to write in the function description, so it might be useful to read the full version, not only my interpretation ;)

Reply:
Ok, I'll fix it according to your comment! :)

Reply:
Thank you!